src/shared/pretty-print.c

```diff
@@ -170,13 +189,13 @@ int cat_files(const char *file, char **dropins, CatFlags flags) {
         int r;
 
         if (file) {
-                r = cat_file(file, false);
+                r = cat_file(file, /* newline= */ false, flags);
```

Comment:
still think it would be nicer to just make the bool a flag and not make this needlessly complex by making this two params. but also doesn't matter